packages/bundler-sdk-viem/test/sharedLiquidity.test.ts

```diff
@@ -0,0 +1,803 @@
+import {
+  ChainId,
+  DEFAULT_SLIPPAGE_TOLERANCE,
+  MathLib,
+  NATIVE_ADDRESS,
+  addresses,
+} from "@morpho-org/blue-sdk";
+import { metaMorphoAbi, publicAllocatorAbi } from "@morpho-org/blue-sdk-viem";
+import { markets, vaults } from "@morpho-org/morpho-test";
+import { useSimulationState } from "@morpho-org/simulation-sdk-wagmi";
+import { renderHook, waitFor } from "@morpho-org/test-wagmi";
+import { configure } from "@testing-library/dom";
+import dotenv from "dotenv";
+import { parseEther, parseUnits } from "viem";
+import { describe, expect } from "vitest";
+import { donator, setupBundle } from "./helpers.js";
+import { test } from "./setup.js";
+
+dotenv.config();
```

Comment:
This should not be necessary with `vitest` (populateBundle.test.ts does not have this)